### ISSUE TITLE
Add Hugging Face data importer

### ISSUE THREAD

**Issue by eu9ene** [mozilla/firefox-translations-training#399]
Apparently we already use some monolingual data from there as a custom corpus based on @gregtatum's investigation. Also we have a tool to list the available data https://github.com/mozilla/firefox-translations-training/pull/397

**Comment by gregtatum** [comment]
We'll need to be careful we don't open the pipeline up to arbitrary python execution. I noticed a few warnings while navigating huggingfaces about this, but I haven't looked into it.

**Comment by eu9ene** [comment]
Agreed. Just a note that we already download bicleaner-ai models from hugging face using `bicleaner-ai-download` tool that uses their lib to pull the data.
